tilequeue/process.py

```diff
@@ -443,3 +443,104 @@ def process_coord(coord, nominal_zoom, feature_layers, post_process_data,
         unpadded_bounds, cut_coords, buffer_cfg, extra_data, scale)
 
     return all_formatted_tiles, extra_data
+
+
+def convert_source_data_to_feature_layers(rows, layer_data, bounds, zoom):
+    # TODO we might want to fold in the other processing into this
```

Comment:
This would have been a performance improvement, if we find that when we run it on dev it's too slow. If it's unnecessary, it might be better to hold off on this since it might change with phase 3 anyway.